core/persephone/workflow.py

```
@@ -0,0 +1,393 @@
+            f"/_db/{db_name}/_api/document/{cols.edges}/{key}",
+        )
+        return True
+    except ArangoHttpError as e:
+        if e.status_code == 404:
+            return False
+        raise
+
+
+def check_blocked(
+    client: Any,
+    db_name: str,
```

Ruff (0.15.0)

[warning] Unused function argument: `db_name`

(ARG001)